shared-bindings/analogio/AnalogIn.c

```diff
@@ -36,27 +36,25 @@
 #include "shared-bindings/analogio/AnalogIn.h"
 #include "shared-bindings/util.h"
 
-//| .. currentmodule:: analogio
+//|class AnalogIn:
```

Comment:
Can this be `//| class`?  My editor will want to prefix all lines with `//| ` in one of these regions, and also it means that the first character of `class` will be lined up like a 4-spaces-indent of standard python code.

Reply:
Yes, there should be a space included in this prefix. The validation script I gave @dherrada yesterday should verify it.

Reply:
When I add a space before class, it says invalid syntax